udapi/core/tests/external_tests.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-udapy read.Conllu filename=UD_Czech_sample.conllu write.Conllu > out.conllu && diff UD_Czech_sample.conllu out.conllu
+udapy read.Conllu filename=data/UD_Czech_sample.conllu write.Conllu > out.conllu && diff data/UD_Czech_sample.conllu out.conllu
```

Comment:
But the file is still here https://github.com/udapi/udapi-python/blob/master/udapi/core/tests/UD_Czech_sample.conllu